src/AdvancedLogger.ts

```diff
@@ -73,10 +73,7 @@ export default class AdvancedLogger<T> {
 
             this.service
                 .sendAllLogs(logs)
-                .catch(error => {
-                    console.log(error);
-                    // todo Retry sending logs here or in the service
-                });
+                .catch(error => console.error(error));
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Calls to 'console.error' are not allowed. ([typescript:S2228](https://sonarcloud.io/organizations/alexeypopovua-github/rules#rule_key=typescript%3AS2228))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=advanced-logger&pullRequest=75&issues=AWfltDhort4Kct_4x7pq&open=AWfltDhort4Kct_4x7pq)

[ISSUE_KEY]: AWfltDhort4Kct_4x7pq